.github/workflows/test.yml

```diff
@@ -88,6 +88,7 @@ jobs:
           cargo test --release --workspace \
           --all-features \
           --all-targets \
+          -- --include-ignored
```

Comment:
I set all optimism tests as `#[ignore]` because I did not have the required dependencies installed.